spec/support/helpers.rb

```diff
@@ -195,7 +196,7 @@ def install_gemfile(*args)
       gemfile(*args)
       opts = args.last.is_a?(Hash) ? args.last : {}
       opts[:retry] ||= 0
-      bundle :install, opts
+      bundle :install, opts.merge(:suppress_stderr => true)
```

Comment:
Ah, this would suppress every call of `install_gemfile` in every spec. Removing that for sure.